### ISSUE TITLE
Location fingerprinting

### ISSUE THREAD

**Issue by jonathanstark** [google/physical-web#15]
Not saying this is a good thing or bad thing, but it occurs to me that if the beacons became fairly widespread, it would be easy for any application that had access to Bluetooth to fingerprint a geolocation.

For example, let's say the Chrome browser on Android had access to Bluetooth. It could then detect the unique combination of URLs being broadcast near me and use that as a fingerprint to tell who else was near that same "beacon fingerprint".

Yes, this could probably also be done with GPS location data but that's a different permission and it's unlikely that someone who was allowing Bluetooth access to an application would consider that would also pinpoint their geolocation. 

In other words, if we wanted to control access to our location, we would also have to turn off Bluetooth.

Now that I'm thinking of it, it occurs to me that simple Google search results could be modified based on the beacons that were near me. So if I search for "car wash" and I happen to be physically near a car wash that's broadcasting, it might become the first search ranking results in the organic Google search results because of the physical proximity.

 Again I'm not saying this is good or bad but it's an interesting consideration.
